packages/api/endpoints/collections.js

```diff
@@ -43,12 +44,10 @@ const log = new Logger({ sender: '@cumulus/api/collections' });
  * @returns {Promise<Object>} the promise of express response object
  */
 async function list(req, res) {
+  // eslint-disable-next-line no-unused-vars
   const { getMMT, includeStats, ...queryStringParameters } = req.query;
-  const collection = new Collection(
-    { queryStringParameters },
-    undefined,
-    process.env.ES_INDEX,
-    includeStats === 'true'
+  const collection = new CollectionSearch(
+    { queryStringParameters }
   );
   let result = await collection.query();
```

Comment:
So Jenny's comments mentioned keeping `includeStats`, and also if it is true it should still use ES for that. I'm not exactly sure what that should look like, but that doesn't seem to be happening here.